### PR TITLE
README-added Pacman installation. Err handling.

### PR DESCRIPTION
**Pull request by ivankonevv**
Installation instructions and a description of pacman are written. Handling some errors.

### CODE REVIEW

File: cx/op_http.go [hunk 1]
```diff
@@ -127,7 +127,10 @@ func opHTTPHandle(prgrm *CXProgram) {
 		copy(i2, PROGRAM.Memory[i2Off:i2Off+i2Size])
 
 		PROGRAM.Callback(handlerFn, [][]byte{i1, i2})
-		fmt.Fprint(w, ReadStr(callFP, handlerFn.Inputs[0]))
+		_, err := fmt.Fprint(w, ReadStr(callFP, handlerFn.Inputs[0]))
+		if err != nil {
+			fmt.Print("http.HandleFunc, Fprint failed")
```
Comment (jdknives) [inline cx/op_http.go:132]: Would we want to print the error here as well? Can we format the error message properly here please?  Maybe something along the lines of:

```go
fmt.Printf("http.HandleFunc: printing to responsewriter failed: %w \n", err)
```

File: cmd/cx/main.go [hunk 2]
```diff
@@ -322,15 +322,23 @@ func checkCXPathSet(options cxCmdFlags) {
 
 	// Creating directories in case they do not exist.
 	if _, err := cxcore.CXStatFile(CXPATH); os.IsNotExist(err) {
-		cxcore.CXMkdirAll(CXPATH, 0755)
+		if err := cxcore.CXMkdirAll(CXPATH, 0755); err != nil {
+			fmt.Println("cxgo/main.go: make CXPATH directory failed")
```
Comment (jdknives) [inline cmd/cx/main.go:326]: Probably a good idea to print the error here as well. Applies to the other instances of `fmt.Println` below as well.

File: README.md [hunk 3]
```diff
@@ -133,9 +133,27 @@ make test
...
+It uses simple compressed files as a package format, and maintains a text-based package database.
 
-Before compiling CX, install dependencies with `pacman` (download MSYS2):
+How to install pacman?      
+
+Go to MSYS2 [official site](https://www.msys2.org) and click to download link.  
+Run the installer, when done tick **Run MSYS2 now**     
+Update the package database and base packages.      
+```
+pacman -Syu
```
Comment (jdknives) [inline README.md:147]: If we add this here, we can get rid of the `pacman -Sy` command a few lines down.

File: README.md [hunk 4]
```diff
@@ -133,9 +133,27 @@ make test
...
+```
+Run "MSYS2 MSYS" from Start menu. Update the rest of the base packages.
```
Comment (jdknives) [inline README.md:149]: Is it necessary to open up MYSYS again after the first pacman command?

File: README.md [hunk 5]
```diff
@@ -133,9 +133,27 @@ make test
...
+```
+pacman -Su
+```
+Done
```
Comment (jdknives) [inline README.md:153]: Redundant comment.

File: README.md [hunk 6]
```diff
@@ -133,9 +133,27 @@ make test
 
 ### Compiling CX on Windows
 
-Compiling CX on Windows requires a recent version of Git to be installed. 
+Compiling CX on Windows requires a recent version of Git to be installed.
```
Comment (jdknives) [inline README.md:136]: I would shorten this whole bit to something like: 

```
Compiling CX on Windows requires Git and Pacman to be installed. 

In order to install Pacman, visit [MYSYS2](example.xyz) <-- put the actual link to the download section here
and download the latest version of MYSYS2. After installing pacman, chose `Run MYSYS2 now`. 

Update the package databse and base packages with:

pacman -Syu

Comment (jdknives) [reply README.md:136]: Can we install Git with pacman?

**Review by jdknives — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (jdknives) [reply README.md:136]: Both `mingw` as well as `git` can be installed with `pacman` so we should probably do it like that.

File: README.md (hunk 4 above)
Comment (archerixx) [reply README.md:149]: No. You can run multiple command while MSYS is up. After install, MSYS can be opened autoamtically (option that comes as part of setup)

File: README.md (hunk 6 above)
Comment (SkycoinSynth) [reply README.md:136]: Yes

**Comment by SkycoinSynth** [comment]
Ivan can you change, update those and send updated pull request ASAP. So I can merge it in.

**Pull request closed without merging**